docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -27,7 +27,7 @@ The main media controller features include:
 
  - Media Control Metadata Properties
 
-     You can [set the metadata](#media-control-metadata-properties) on the server side, and then get that metadat on the client side.
+     You can [set the metadata](#media-control-metadata-properties) on the server side, and then get that metadata on the client side.
 
 ## Prerequisites
 
```

Comment:
In **line no: 38 and 45**:
It is recommended to add a colon (:) at the end of the sentence as it is followed by a code block.

For example, 
"To use the [Tizen.Multimedia.Remoting.MediaController](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaController.html) class:"


If this is not in your scope to change, please ignore my comments.

Reply:
Done